Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorSmoothingEffect.py

```diff
@@ -78,6 +78,11 @@ def setupOptionsFrame(self):
     self.jointTaubinSmoothingFactorSlider.pageStep = 0.1
     self.jointTaubinSmoothingFactorLabel = self.scriptedEffect.addLabeledOptionsWidget("Smoothing factor:", self.jointTaubinSmoothingFactorSlider)
 
+    self.applyToAllSegmentsCheckBox = qt.QCheckBox()
+    self.applyToAllSegmentsCheckBox.setToolTip("Apply smoothing effect to all visible segments in this segmentation node. This operation may take a while.")
+    self.applyToAllSegmentsCheckBox.objectName = self.__class__.__name__ + 'ApplyToAllVisibleSegments'
+    self.applyToAllSegmentsMMLabel = self.scriptedEffect.addLabeledOptionsWidget("Apply to all segments:", self.applyToAllSegmentsCheckBox)
```

Comment:
Remove MM. It refers to millimeter (physical distance unit) - therefore it is not relevant for this variable's name.